multi-casskop/documentation/version-alpha.md

```diff
@@ -0,0 +1,123 @@
+# Alpha Documentation
+
+This is an early work on documentation not yet well structured, things I note here at this time :)
+
+## What is Multi-CassKop
+
+Multi-CassKop is a new Kubernetes operator that sits above CassKop operator.
+CassKop allows to create a Cassandra cluster in a Kubernetes cluster, using the custom ressource **CassandraCluster**.
+
+Multi-CassKop goal is to bring the ability to deploy a Cassandra cluster within different regions, each of them running
+an independant Kubernetes cluster.
+
+Multi-CassKop will be talking to several Kubernetes clusters APIs, and is managing a new custom ressource named
+**MultiCasskop**.
+
+The only goal of Multi-CassKop is to create **CassandraCluster** ressources in each of k8s clusters, where there is a
+locally instance of CassKop that will deploy Cassandra cluster according to the required definition.
+
+Multi-Casskop insure that the Cassandra nodes deployed by each local CassKop will be part of the same Cassandra ring by
+managing a coherent creation of CassandraCluster objects from it's own MultiCasskop custom ressource.
+
+## Software Organisation
+
+This operator uses: 
+- Operator sdk to setup the new CRD `MultiCasskop`.
+- Admiralty's [multicluster-controller](https://github.com/admiraltyio/multicluster-controller) to allow talking to
+  several kubernetes api servers.
+  
+
+## Launching MultiCassKop operator
+
+MultiCassKop take into parameters a list of kubernetes contexts names as defined in the KUBECONFIG file.
+
+example: 
+```
+./multiCassKop dex-sallamand-kaas-prod-priv-sph dex-sallamand-kaas-prod-priv-bgl
+```
+
+>**Note:** MultiCassKop will ONLY uses the first kubernetes passed as parameter to MultiCasskop objects
+
+# How MultiCassKop works
+
+MultiCassKop starts by iterrating on every contexts passed in parameters then it register the controller. 
+The controller needs to be able to interract with MultiCasskop and CassandraCluster CRD objetcs.
+In addition the controller needs to watch for MultiCasskop as it will need to react on any changes occurs on
```

Comment:
occurs -> that occurred

Reply:
oh forgot about this file, i'll read it again to see what need to be changed